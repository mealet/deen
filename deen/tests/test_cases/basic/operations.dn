fn main() i32 {
  println!("{}" 25 + 5);
  println!("{}" 25 - 5);
  println!("{}" 25 * 5);
  println!("{}" 25 / 5);
  println!("{}" 25 % 5);

  return 0;
}
