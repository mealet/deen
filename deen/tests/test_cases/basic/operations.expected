30
20
125
5
0
0
800
